packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.js

```diff
@@ -75,6 +71,13 @@ export default class QuanticRefineToggle extends LightningElement {
    * @type {string}
    */
   @api title = this.labels.sortAndFilters;
+  /**
+   * Indicates whether to disable the facet ordering feature according to [the dynamic navigation experience](https://docs.coveo.com/en/3383/leverage-machine-learning/about-dynamic-navigation-experience-dne).
+   * @api
+   * @type {boolean}
+   * @defaultValue `false`
+   */
+  @api disableFacetOrdering = false;
```

Comment:
This should be called `disableDynamicNavigation` to be consistent everywhere.

Reply:
the reason we named it that way was to be more specific since the DNE includes more features than facetOrdering 
[see here](https://github.com/coveo/ui-kit/pull/3000#:~:text=Since%20the%20DNE,disableFacetOrdering%20for%20example)